src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -67,8 +67,11 @@ import { getChartProperties } from './highcharts/helpers';
 import { isDataOfReasonableSize } from './highChartsCreators';
 import { NORMAL_STACK, PERCENT_STACK } from './highcharts/getOptionalStackingConfiguration';
 
-const enableAreaChartStacking = (stacking: any) => {
-    return stacking || isUndefined(stacking);
+const enableAreaChartStacking = (stackMeasures: boolean, stacking: any) => {
```

Comment:
In order to fulfil our unwritten rule (leave code in better state than it was before) please find the right type for `stacking` argument instead of `any` :) 

also `enableAreaChartStacking` indicates action that you want to enable stacking which is not true :)

`isStackingEnabled` would be much precise :) do you agree ? :)